### PR TITLE
Include product filter to separate common metrics by product

### PR DESCRIPTION
**Pull request by Wil312**
Added a jmx-exporter config file to label common metrics with a product filter. Updated dashboard accordingly.

### CODE REVIEW

File: ita-dash.json [hunk 1]
```diff
@@ -1,1834 +1,1752 @@
...
-                {
-                  "color": "red",
-                  "value": 80
-                }
-              ]
+      "pluginVersion": "8.2.5",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "java_lang_OperatingSystem_AvailableProcessors{instance=\"localhost:2991\"}",
```
Comment (bcarnevale) [inline ita-dash.json]: There is still an instance reference here. Is that intentional?

File: ita-dash.json [hunk 2]
```diff
@@ -1,1834 +1,1752 @@
...
+          "legendFormat": "{{name}} count ({{instance}})",
+          "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "exemplar": true,
+          "expr": "increase(Catalina_GlobalRequestProcessor_errorCount{instance=~\"10.226.27.21:8099|10.226.29.166:8099\"}[10m])",
```
Comment (bcarnevale) [inline ita-dash.json]: Is this also intentional?

File: ita-dash.json [hunk 3]
```diff
@@ -1,1834 +1,1752 @@
...
-            "color": {
-              "mode": "palette-classic"
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "exemplar": true,
+          "expr": "(Catalina_ThreadPool_currentThreadsBusy{instance=~\"10.226.27.21:8099|10.226.29.166:8099\"} / Catalina_ThreadPool_maxThreads{instance=~\"10.226.27.21:8099|10.226.29.166:8099\"}) * 100",
```
Comment (bcarnevale) [inline ita-dash.json]: Also here?

**Review by atl-mk — APPROVED** [review]

**Comment by atl-mk** [comment]
Be wary that you should make the changes on the bulldog grafan instance, they may get lost otherwise

File: ita-dash.json (hunk 3 above)
Comment (Wil312) [reply ita-dash.json]: 🦅 Well spotted! I've removed this.

File: ita-dash.json (hunk 2 above)
Comment (Wil312) [reply ita-dash.json]: 🦅 Well spotted! I've removed this.

File: ita-dash.json (hunk 1 above)
Comment (Wil312) [reply ita-dash.json]: 🦅 Well spotted! I've removed this.

**Pull request merged**